AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+Model.swift

```diff
@@ -64,10 +80,11 @@ extension GraphQLRequest {
     public static func query<M: Model>(from modelType: M.Type,
                                        where predicate: QueryPredicate? = nil) -> GraphQLRequest<[M]> {
         let document = GraphQLListQuery(from: modelType, predicate: predicate)
+        // TODO: decodePath should just be document.name and return `GraphQLRequest<GraphQLListResponse<M>>`
         return GraphQLRequest<[M]>(document: document.stringValue,
                                    variables: document.variables,
-                                   responseType: [M].self,
-                                   decodePath: document.decodePath)
+                                   responseType: [M].self, // TODO: should be `GraphQLListResponse<M>`
```

Comment:
If we don't already have one let's get an issue filed for this so we can refer to it in the release notes/commit logs when we release this incremental update

Reply:
added https://github.com/aws-amplify/amplify-ios/issues/291